coalib/results/TextPosition.py

```diff
@@ -33,6 +33,19 @@ def column(self):
         return self._column
 
     @classmethod
+    def calc_position(self, text, line, column):
+        """
+        Calculates position in a string from line number and column.
+        Uses \\n  as the newline character. Position starts from 0
+        """
+        lines = text.splitlines(keepends=True)
```

Comment:
@AbdealiJK  this is a different algo? that was for calculating line, col from positionn, this ones the other way round ? i dont understand